src/System.Net.Requests/tests/GlobalProxySelectionTest.cs

```diff
@@ -38,6 +38,7 @@ public bool IsBypassed(Uri host)
             }
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17842")]
```

Comment:
>For consistency we should only use // TODO: #...

I've seen the issue number like this (in the SkipOnTargetFramework) in other libraries in CoreFx.